packages/cli/tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "src",
+    "rootDir": "",
```

Comment:
@thymikee can you help me understand what this file is doing? Had to change it so that `../package.json` require works. 

This doesn't seem to be affecting the directory structure inside `outDir`, as it's still flat (`build/index.js` is present, it's not `build/src/index.js` or something).

Reply:
It's an indication that we keep all of our source files inside src/ dir. Likely makes the compilation a bit faster. For the package.json include, I'd go with `"include": ["../package.json"]`